packages/office-ui-fabric-react/src/components/pickers/BasePicker.tsx

```diff
@@ -154,6 +154,7 @@ export class BasePicker<T, P extends IBasePickerProps<T>> extends BaseComponent<
                 className={ css('ms-BasePicker-input', styles.pickerInput) }
                 ref={ this._resolveRef('input') }
                 onFocus={ this.onInputFocus }
+                onBlur={ this.onInputBlur }
```

Comment:
Great feedback! This makes me wonder about 2 questions:
(1) what's the convention when we use native onBlur and when we use "inputProps"?  TextField uses native while Pickers use "inputProps".
(2) for onFocus, we add inputProps.onFocus and another onFocus={ this.onInputFocus }, which one takes precedence?  Or both will get called?